src/modules/ekf2/EKF/python/tuning_tools/mc_wind_estimator/readme.md

```diff
@@ -1,22 +1,27 @@
 # PX4 Drag fusion parameter tuning algorithm
+
 In PX4, drag fusion can be enabled in order to estimate the wind when flying a multirotor, assuming that the body vertical acceleration is produced by the rotors and that the lateral forces are produced by drag.
 The model assumes a combination of:
-1. momentum drag: created by the rotors changing the direction of the incoming air, linear to the relative airspeed. Parameter `EKF2_MCOEF`
-2. bluff body drag: created by the wetted area of the aircraft, quadratic to the relative airspeed. Parameters `EKF2_BCOEF_X` and `EKF2_BCOEF_Y`
+1. **momentum drag:** created by the rotors changing the direction of the incoming air, linear to the relative airspeed.
+   Parameter [`EKF2_MCOEF`](https://docs.px4.io/master/en/advanced_config/parameter_reference.html#EKF2_MCOEF)
+2. **bluff body drag:** created by the [wetted area](https://en.wikipedia.org/wiki/Wetted_area) of the aircraft, quadratic to the relative airspeed.
+   Parameters [`EKF2_BCOEF_X`](https://docs.px4.io/master/en/advanced_config/parameter_reference.html#EKF2_BCOEF_X) and [`EKF2_BCOEF_Y`](https://docs.px4.io/master/en/advanced_config/parameter_reference.html#EKF2_BCOEF_Y)
 
 The python script was created to automate the tuning of the aforementioned parameters using flight test data.
 
 ## How to use this script
+
 First, a flight log with enough information is required in order to accurately estimate the parameters.
 The best way to do this is to fly the drone in altitude mode, accelerate to a moderate-high speed and let the drone slow-down by its own drag.
 Repeat the same maneuver in all directions and several times to obtain a good dataset.
 
-/!\ NOTE: the current state of this script assumes no wind. Some modifications are required to estimate both the wind and the parameters at the same time.
+> **NOTE:** the current state of this script assumes no wind. Some modifications are required to estimate both the wind and the parameters at the same time.
 
 Then, install the required python packages:
 ```
 pip install -r requirements.txt
 ```
+
 and run the script and give it the log file as an argument:
 ```
 python drag_replay.py <logfilename.ulg>
```

Comment:
Neither of the scripts in the folder are called `drag_replay.py`.

The scripts in this folder are:
- [drag_fusion_symbolic.py](https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/ekf2/EKF/python/tuning_tools/mc_wind_estimator/drag_fusion_symbolic.py)
- [mc_wind_estimator_tuning.py](https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/ekf2/EKF/python/tuning_tools/mc_wind_estimator/mc_wind_estimator_tuning.py)

So which do we call as shown above, and what does the other one do?

Reply:
Right, `mc_wind_estimator_tuning.py` is the one to use.
`[drag_fusion_symbolic.py](https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/ekf2/EKF/python/tuning_tools/mc_wind_estimator/drag_fusion_symbolic.py) is just the derivation of the drag fusion (for wind estimator) equations used in the code, they are not directly relevant for tuning and I should probably move it from here to the ekf2 derivation directly